### PR TITLE
Changelog for v0.15.0-rc1

### PR DESCRIPTION
**Pull request by 6543**
as title

please make suggestions or push against to impruve

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,170 @@
-## [0.14.3](https://github.com/woodpecker-ci/woodpecker/releases/tag/v0.14.3) - 2021-10-30
+## [v0.15.0](https://github.com/woodpecker-ci/woodpecker/releases/tag/v0.15.0) - 2022-01-08
```
Comment (6543) [inline CHANGELOG.md:1]:
```suggestion
## [v0.15.0-rc1](https://github.com/woodpecker-ci/woodpecker/releases/tag/v0.15.0-rc1) - 2022-01-08
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,172 @@
...
+  * Show yml config of pipeline in UI (#649)
+  * Multiarch build for cli and agent docker images (#634), (#622)
+  * Get secrets in settings (#604)
+  * Add multi-pipeline support to exec & lint (#568)
+  * Add repo branches endpoint (#481)
+  * Add repo permission endpoint (#436)
+  * Add web-config endpoint (#433)
+  * Replace www-path with www-proxy option for development (#248)
+* SECURITY
+  * Add linter bidichk to prevent malicios utf8 chars (#516)
```
Comment (jolheiser) [inline CHANGELOG.md:25]:
```suggestion
  * Add linter bidichk to prevent malicious utf8 chars (#516)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,172 @@
...
+* BUGFIXES
+  * Remove "panic()" as much as posible from code (#682)
```
Comment (jolheiser) [inline CHANGELOG.md:27]:
```suggestion
  * Remove "panic()" as much as possible from code (#682)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,172 @@
...
+* ENHANCEMENTS
+  * Build ref link point to commit not compare if only one commit was pushed (#673)
+  * Hide multi line secrets from log (#671)
+  * Do not exclude repo owner from gated rule (#641)
+  * Add field for image list in Secrets Repo Settings (Web UI) (#638)
+  * Use Woodpecker theme colors on Safari Tab Bar / Header Bar (#632)
+  * Add "woodpeckerci/plugin-docker-buildx" to privileged plugins (#623)
+  * Use gitlab generic webhooks instead of drone-ci-service (#620)
+  * Calculate build number on creation (#615)
+  * Hide gin routes logging on none-debug starts (#603)
```
Comment (jolheiser) [inline CHANGELOG.md:55]:
```suggestion
  * Hide gin routes logging on non-debug starts (#603)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,172 @@
...
+  * Update proxy docs (#573)
+  * Update Documentation (fix diffs and add settings) (#569)
+  * Update README (#560)
+  * Add notice of supported YAML versions in docs (#556)
+  * Update docs dependencies (#553)
+  * Update Agent and Pipeline syntax documentation (#506)
+  * Update docs about selecting agent based on platform (#470)
+  * Add plugin marketplace (for official plugins) (#451)
+  * Improve docs (#450)
+  * add search to docs (#448)
```
Comment (jolheiser) [inline CHANGELOG.md:137]:
```suggestion
  * Add search to docs (#448)
```

**Review by jolheiser — CHANGES_REQUESTED** [review]
Small nits

**Review by jolheiser — APPROVED** [review]

**Pull request merged**